src/packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -1311,6 +1369,35 @@ generator client {
             } else {
               acc[key] = value
             }
+            if (key === 'count') {
+              if (typeof value === 'object' && value) {
+                acc.select[key] = { select: mapAllCount(value) }
+                unpacker = (data) => {
+                  if (Array.isArray(data)) {
+                    data = data.map(row => {
+                      if (row.count) {
+                          row.count = mapAllCount(row.count)
+                      }
+                      return row
+                    })
+                  }
+                  return data
+                }
+              } else if(typeof value === 'boolean') {
+                acc.select[key] = { select: { $all: value } }
+                unpacker = (data) => {
+                  if (Array.isArray(data)) {
+                    data = data.map(row => {
+                      if (typeof row.count === 'object' && row.count?._all) {
```

Comment:
```suggestion
                      if (row && typeof row.count === 'object' && row.count?._all) {
```

Because `typeof null === "object"`